cves/CVE-2015-5964.yml

```diff
@@ -246,37 +248,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: having a secondary system keep track of memory being allocated could have automatically fixed this vulnerability.
```

Comment:
I'm not sure if this problem is directly related to memory or rather the session store specifically, which may have its own limits.

Reply:
That's a good point, I'm not too up on how sessions are stored server side